README.md

```diff
@@ -1,7 +1,7 @@
 Ember-load-initializers [![Build Status](https://travis-ci.org/ember-cli/ember-load-initializers.svg?branch=master)](https://travis-ci.org/ember-cli/ember-load-initializers)
 ==============================================================================
 
-A tiny add-on to autoload your initializer files in ember-cli.
+A tiny add-on to autoload your initializer and instance initializer files in ember-cli. This add-on simply iterates over files inside `app/initializers` and `app/instance-initializers` and invokes the `app.initializer` and `app.instanceInitializer` methods respectively, by passing on the resolved name of the files.
```

Comment:
I suggest removing the word `simply`.
https://css-tricks.com/words-avoid-educational-writing/#article-header-id-2